### ISSUE TITLE
A11y - Global

### ISSUE THREAD

**Issue by villivald** [villivald/villivald.com#31]
- [x] Language switch - Form elements do not have associated labels
- [x] Menu button (hamburger) does not have accessible name
- [x] Html lang attribute
- [x] Skip to the main content (grid) button
- [x] Skip to contact footer
- [x] Focus indicators - check and fix
     - [x] Main Page
     - [x] About
     - [x] Blog
     - [x] Contact
     - [x] Books / Statistics
     - [x] Projects
     - [x] Study
     - [x] Uses
     - [x] Old
